timetracker/entries/forms.py

```diff
@@ -44,6 +46,10 @@ def clean(self):
         start = cleaned_data.get('start', None)
         end = cleaned_data.get('end', None)
 
+        if 'end_now' in self.data:
+            cleaned_data['stop']= timezone.now()
+
```

Comment:
Generally it's best practice not to change any data in the a form's clean() method. It would be better here to define a clean_stop() method, similar to clean_start that sets the timezone.now() value.